RF24Gateway.cpp

```diff
@@ -207,15 +206,15 @@ int ESBGateway<mesh_t, network_t, radio_t>::allocateTunDevice(char* dev, int fla
 
     // Create device
     if (ioctl(fd, TUNSETIFF, (void*)&ifr) < 0) {
-        //close(fd);
+        // close(fd);
```

Comment:
Are all these comment changes from clang-format? I thought I configured it to allow for any spaces after single line comments.

> [!TIP]
> Use the .clang-format config `file` automatically (with optional glob file syntax):
> ```
> clang-format-14 -i -style=file RF24*.*
> ```
> The `-i` modifies files in-place instead of piping changes to stdout.

Reply:
Yup, I just formatted using clang-format 14.0.6 and it did this. I assumed it just picks up the style file in RF24Gateway folder automatically.

Reply:
It might've in v12 before, but there are significant (and rather un-noticable) changes between major versions of clang. I think without the `-style=file`, clang-format defaults to `llvm` style (which is super strict by comparison to our config).

Reply:
Lucky that comment changes were practically all that it wanted to do.

BTW, github won't let us (at cpp-linter project) make a suggestion to remove lines from a diff. That's why those suggestions seems lacking.